MTM-46453 Added requirements for all major cockpit features. (#532)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/reports.md

Reports enable you to track applications, alarms, assets, and other data in a dashboard layout. Reports are global dashboard pages, regardless of the asset hierarchy.

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

- To view reports: READ permission for permission type "Inventory"

> - To edit reports: ADMIN permission for permission type "Inventory"

Reviewer: ```suggestion
- To edit a report: ADMIN permission for permission type "Inventory"
```

- To add a new report: CREATE permission for permission type "Inventory"
- To delete a report: ADMIN permission for permission type "Inventory"

> - Without READ permission for permission type "Inventory", no measurements will be shown within the report.

Reviewer: ```suggestion

Without READ permission for permission type "Inventory", no measurements will be shown within the report.
```

Reports enable you to track applications, alarms, assets, and other data in a dashboard layout. Reports are global dashboard pages, regardless of the asset hierarchy.

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

- To view reports: READ permission for permission type "Inventory"
- To edit reports: ADMIN permission for permission type "Inventory"
- To add a new report: CREATE permission for permission type "Inventory"
- To delete a report: ADMIN permission for permission type "Inventory"

> Without READ permission for permission type "Inventory", no measurements will be shown within the report.

Author: removed it from this place.

Reports enable you to track applications, alarms, assets, and other data in a dashboard layout. Reports are global dashboard pages, regardless of the asset hierarchy.

{{< c8y-admon-req >}}

> ROLES & PERMISSIONS:

Reviewer: How about the microservice subscription?

Author: When I created a new user I only assigned READ or ADMIN permissions for the inventory and it was enough to handle reports.

Author: I guess it might be required on a tenant level, should I mention that?

Reviewer: Yes, I guess the tenant must be subscribed to this microservice?

Author: Hmm, in the end I guess you don't need any microservice subscriptions in order to access the reports.

- To view reports: READ permission for permission type "Inventory"
- To edit reports: ADMIN permission for permission type "Inventory"
- To add a new report: CREATE permission for permission type "Inventory"
- To remove a report: ADMIN permission for permission type "Inventory"

>   {{< /c8y-admon-req >}}

Reviewer 2: ```suggestion
{{< /c8y-admon-req >}}
```

Author: changed

>   {{< c8y-admon-info >}}

Reviewer 2: ```suggestion
{{< c8y-admon-info >}}
```

Author: fixed

>   Without READ permission for permission type "Inventory", no measurements will be shown within the report.

Reviewer: I would probably add this to the requirements section instead of showing a separate info box here. In general, @MWindel you should scan through the tests to see, if inline more requirements are mention which could be deleted or moved.

>   {{< /c8y-admon-info >}}

Reviewer 2: ```suggestion
{{< /c8y-admon-info >}}
```
